grep-printer/src/summary.rs

```diff
@@ -261,13 +261,13 @@ impl SummaryBuilder {
         self
     }
 
-    /// Exclude count-related summary results with no matches.
+    /// Include count-related summary results with no matches.
     ///
     /// When enabled and the mode is either `Count` or `CountMatches`, then
-    /// results are not printed if no matches were found. Otherwise, every
-    /// search prints a result with a possibly `0` number of matches.
-    pub fn exclude_zero(&mut self, yes: bool) -> &mut SummaryBuilder {
-        self.config.exclude_zero = yes;
+    /// results are printed even if no matches were found. Otherwise, every
+    /// search only prints a result if there were matches.
+    pub fn include_zero(&mut self, yes: bool) -> &mut SummaryBuilder {
```

Comment:
I'm not following why this change was made. It's a breaking API change and seems purely cosmetic.